README.md

```diff
@@ -0,0 +1,115 @@
+# TextPaddleEncoder
+
+**TextPaddleEncoder** is a class that wraps the text embedding functionality from the **PaddlePaddle** and **PaddleHub**.
+
+
+`TextPaddleEncoder` encode text stored in the `text` attribute of the [**Document**](https://github.com/jina-ai/jina/blob/master/.github/2.0/cookbooks/Document.md) and saves the encoding in the embedding attribute.
+
+
+
+## Prerequisites
+
+None
+
+
+## Usages
+
+### Via JinaHub (🚧W.I.P.)
+
+Use the prebuilt images from JinaHub in your python codes, 
+
+```python
+from jina import Flow
+	
+f = Flow().add(
+        uses='jinahub+docker://TextPaddleEncoder',
+        volumes='/your_home_folder/.cache/paddle:/root/.cache/paddle')
+```
+
+or in the `.yml` config.
+	
+```yaml
+jtype: Flow
+pods:
+  - name: encoder
+    uses: 'jinahub+docker://TextPaddleEncoder'
+    volumes: '/your_home_folder/.cache/paddle:/root/.cache/paddle'
```

Comment:
please make sure this is actually used. This is in the clip one because it helps. Not sure about here